activiti-cloud-query-service/activiti-cloud-starter-query/src/main/resources/config/query/liquibase/changelog/03-alter.oracle.schema.7.1.0-m10.sql

```diff
@@ -0,0 +1,11 @@
+alter table process_instance
+    add completed_date timestamp;
+
+create index pi_completedDate_idx on process_instance (completed_date);
+
+update process_instance pi
+    set completed_date = (select p_i.last_modified
+                             from process_instance p_i
+                             where p_i.id = pi.id);
```

Comment:
You need to update the where clause to check the status as well. Only process instances in the completed status should have the `completed_date` set.

Reply:
Sure.  I just guarded it to check for both process id, and it's status to be "COMPLETED".